### PR TITLE
Adds controlFlow operator

### PR DESCRIPTION
**Pull request by pavadeli**

### CODE REVIEW

**Comment by pavadeli** [comment]
I will rename it to `filterUpdates`, sounds better, I think. We still need good docs!

File: extensions/sherlock-utils/src/control-flow.spec.ts [hunk 1]
```diff
@@ -0,0 +1,680 @@
...
+                        expect(f$.value).to.equal('c');
+                    });
+
+                    includeUnresolved
+                        ? shouldNotHaveReacted(unresolved)
+                        : shouldHaveReactedOnce('c');
+                });
+            });
+
+            describe('skipFirst', () => {
```
Comment (njirem) [inline extensions/sherlock-utils/src/control-flow.spec.ts]: Should `skipFirst` not also consider a transaction as 'one big update'?

File: src/reactor/reactor.ts [hunk 2]
```diff
@@ -239,10 +239,10 @@ export class Reactor<V> implements Observer {
                 skipFirst = false;
             } else if (once) {
                 stopReactors();
-                reaction(value);
+                reaction(value, done);
```
Comment (njirem) [inline src/reactor/reactor.ts]: That won't do much.. But i get it, for consistency. :-P

**Comment by njirem** [comment]
One test fails! :-o

**Pull request merged**